src/strings.rs

```diff
@@ -149,7 +149,7 @@ impl<'a> Borrow<str> for CowStr<'a> {
 }
 
 impl<'a> CowStr<'a> {
-    pub fn to_string(self) -> String {
+    pub fn into_string(self) -> String {
```

Comment:
Agree but note this is a semver break. It's fine if it's scooped into the 0.4 release.

Reply:
I don't think that's the reasoning behind the lint. `to_string` used to be slower because it went through the `fmt::Display` machinery, but that's not relevant here.

The lint that triggered this change seemed to warn against `to_*` methods taking self by value instead of by reference, that's all.